src/components/layout/Layout.tsx

```diff
@@ -101,9 +106,9 @@ export default function ({ children, articles, t, routes }: LayoutProps) {
             </>
           }
           action={
-            <Button id="home-navigation-join-button" onClick={ctx.open}>
+            <JoinUsLink id="home-navigation-join-button" href={discordUrl}>
```

Comment:
**I have here idea**

getDiscordUrl() repeats logic which is already done inside `gatsby-node.js`. We're loading all data connected with pages once, to avoid query repetition and because it takes less time than multiple queries.

Here you can pass simply the link to the discord channel taken from gatsby-config. There is no problem with it. It's main, business context based component so here we can do it.